string label
---
float32 xGoal
float32 yGoal